src/aap_eda/analytics/analytics_collectors.py

```diff
@@ -0,0 +1,422 @@
+)
+def config(since, **kwargs):
+    # TODO:
+    # license_info = get_license()
+    license_info = {}
+
+    install_type = "traditional"
+    if os.environ.get("container") == "oci":
+        install_type = "openshift"
+    elif "KUBERNETES_SERVICE_PORT" in os.environ:
+        install_type = "k8s"
+    return {
+        "platform": {
+            "system": platform.system(),
+            "dist": distro.linux_distribution(),
+            "release": platform.release(),
+            "type": install_type,
+        },
+        # "install_uuid": source.info["install_uuid"],
+        # "tower_url_base": source.info["url"],
+        # "tower_version": source.info["version"],
+        # 'instance_uuid': settings.SYSTEM_UUID,
+        "license_type": license_info.get("license_type", "UNLICENSED"),
+        "free_instances": license_info.get("free_instances", 0),
+        "total_licensed_instances": license_info.get("instance_count", 0),
+        "license_expiry": license_info.get("time_remaining", 0),
+    }
+
+
+@register(
+    "activations_table",
+    "1.0",
+    format="csv",
+    description="Data on activations",
+)
+def activations_table(since, full_path, until, **kwargs):
+    activation_query = """COPY (SELECT core_activation.id,
+       core_activation.name,
+       core_activation.description,
+       core_activation.is_enabled,
+       core_activation.restart_policy,
+       core_activation.restart_count,
+       core_activation.failure_count,
+       core_activation.is_valid,
+       core_activation.rulebook_name,
+       core_activation.rulebook_rulesets,
+       core_activation.ruleset_stats,
+       core_activation.created_at,
+       core_activation.modified_at,
+       core_activation.extra_var,
+       core_activation.decision_environment_id,
+       core_activation.project_id,
+       core_activation.rulebook_id,
+       core_activation.user_id,
+       core_activation.status,
+       core_activation.status_updated_at,
+       core_activation.git_hash,
+       core_activation.status_message,
+       core_activation.latest_instance_id,
+       core_activation.log_level,
+       core_activation.k8s_service_name,
+       core_activation.organization_id,
+       core_activation.source_mappings,
+       core_activation.skip_audit_events
+       FROM core_activation
+       WHERE ((core_activation.created_at > '{0}'
+               AND core_activation.created_at <= '{1}')
+               OR (core_activation.modified_at > '{0}'
+               AND core_activation.modified_at <= '{1}'))
+       ORDER BY core_activation.id ASC) TO STDOUT WITH CSV HEADER
+    """.format(
+        since.isoformat(), until.isoformat()
+    )
+
+    return _copy_table("activations", activation_query, full_path)
+
+
+@register(
+    "audit_rule_table",
+    "1.0",
+    format="csv",
+    description="Data on audit_rules",
+)
+def audit_rules_table(since, full_path, until, **kwargs):
+    activation_instance_ids = models.RulebookProcess.objects.filter(
```

Comment:
I think this query can return duplicated ids. Should we use `distinct`?

Reply:
The ids are the primary keys; they shouldn't have duplicates

Reply:
My point is that you are using OR in the query, so it can return the same id twice if it matches both conditions.

Reply:
added